book/src/proposals/broadmoor.md

```diff
@@ -0,0 +1,568 @@
+        margin-left: auto;
+        margin-right: auto;
+        max-width:800px;
+        /* hack: mdbook doesnt support the full bleed layout we actually want, so just do something that looks decent. */
+        margin-left: 15px;
+    }
+}
+
+.post-subtitle {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 800px;
+    font-style:italic
+}
+
+.post-body {
+    margin-left: auto;
+    margin-right: auto;
+    max-width:800px
+}
+
+.media-box {
+    background-color: #f0f0f0;
+    color: #000;
+    padding-bottom: 16px;
+    margin-bottom: 16px;
+    margin-top:16px
+}
+
+.media-box img {
+    width:100%
+}
+
+.media-box .video-container {
+    width: 100%;
+    position:relative
+}
+
+.media-box .embedded-video, .media-box video {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height:100%
+}
+
+.media-box .chart-header {
+    margin-top: 0;
+    padding-top: 8px;
+    margin-bottom: 8px;
+    margin-left:16px
+}
+
+.media-box img.chart {
+    padding: 8px;
+    width: calc(100% - 16px);
+    object-fit:contain
+}
+
+.media-box p {
+    margin: 8px 16px 0 16px;
+    padding:0
+}
+
+.media-box .photo-attribution {
+    color: #888;
+    font-size:16px
+}
+
+@media (min-width: 569px) {
+    .sidebar-hidden .wrap-two {
+        display: flex;
+        flex-direction: row;
+        justify-content:space-between
+    }
+
+    .sidebar-hidden .wrap-two .media-box:first-child {
+        margin-right: 16px
+    }
+
+    .sidebar-hidden .wrap-two .media-box {
+        width: 50%
+    }
+
+    .sidebar-hidden .wrap-two img {
+        object-fit: cover;
+        max-height:300px
+    }
+}
+
+@media (min-width: 869px) {
+    .sidebar-visible .wrap-two {
+        display: flex;
+        flex-direction: row;
+        justify-content:space-between
+    }
+
+    .sidebar-visible .wrap-two .media-box:first-child {
+        margin-right:16px
+    }
+
+    .sidebar-visible .wrap-two .media-box {
+        width:50%
+    }
+
+    .sidebar-visible .wrap-two img {
+        object-fit: cover;
+        max-height:300px
+    }
+}
+
+
+@media (max-width: 569px) {
+    .sidebar-hidden .media-box {
+        /* hack: mdbook doesnt support the full bleed layout we actually want, so just do something that looks decent. */
+        // left: 50%;
+        // margin-left: -50vw;
+        // margin-right: -50vw;
+        // max-width: 100vw;
+        // position: relative;
+        // right: 50%;
+        // width:100vw
+    }
+}
+
+@media (max-width: 869px) {
+    .sidebar-visible .media-box {
+        /* hack: mdbook doesnt support the full bleed layout we actually want, so just do something that looks decent. */
+        // left: 50%;
+        // margin-left: -50vw;
+        // margin-right: -50vw;
+        // max-width: 100vw;
+        // position: relative;
+        // right: 50%;
+        // width:100vw
+    }
+}
+
+blockquote {
+    border-left: 4px solid #CCC;
+    padding-left: 8px;
+    margin-left: 8px
+}
+
+</style>
+
```

Comment:
So do you want your name on this? :P
How about something like "Written by [Michael Kirk](your site or twitter)
Meet the [A/B Street team](https://a-b-street.github.io/docs/project/team.html)"

Reply:
done!